Content.Server/Disease/DiseaseSystem.cs

```diff
@@ -109,7 +109,9 @@ public override void Update(float frameTime)
                         }
                         foreach (var effect in disease.Effects)
                         {
-                            if (_random.Prob(effect.Probability))
+                            if (_random.Prob(effect.Probability)
+                                && disease.DiseaseSeverity >= effect.MinSeverity
+                                && disease.DiseaseSeverity <= effect.MaxSeverity)
```

Comment:
Isn't prob the most expensive? I assume you'd be doing that last.

Reply:
ah alright , I will change that